<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="wi002648fi.dhcp.missouri.edu" failures="2" tests="20" name="AllTests" time="0.243" errors="0" timestamp="2025-05-01T17:14:03 CDT">
  <testcase classname="edu.mu.examplefortesting.tests.AccountTest" name="testDeposit" time="0.004"/>
  <testcase classname="edu.mu.examplefortesting.tests.AccountTest" name="testDeposit" time="0.001"/>
  <testcase classname="edu.mu.examplefortesting.tests.AccountTest" name="testOverdraft" time="0.0"/>
  <testcase classname="edu.mu.examplefortesting.tests.AccountTest" name="testWithdraw" time="0.0"/>
  <testcase classname="edu.mu.examplefortesting.tests.AccountTest" name="testWithdraw" time="0.0"/>
  <testcase classname="edu.mu.examplefortesting.tests.TransferTest" name="testConcurrentDeposit" time="0.001"/>
  <testcase classname="edu.mu.examplefortesting.tests.TransferTest" name="testConcurrentDeposit" time="0.001"/>
  <testcase classname="edu.mu.examplefortesting.tests.TransferTest" name="testConcurrentDeposit" time="0.0"/>
  <testcase classname="edu.mu.examplefortesting.tests.TransferTest" name="testConcurrentDeposit" time="0.0"/>
  <testcase classname="edu.mu.examplefortesting.tests.TransferTest" name="testConcurrentDeposit" time="0.001"/>
  <testcase classname="edu.mu.examplefortesting.tests.TransferTest" name="testConcurrentDeposit" time="0.0"/>
  <testcase classname="edu.mu.examplefortesting.tests.TransferTest" name="testConcurrentDeposit" time="0.0"/>
  <testcase classname="edu.mu.examplefortesting.tests.TransferTest" name="testConcurrentDeposit" time="0.001"/>
  <testcase classname="edu.mu.examplefortesting.tests.TransferTest" name="testConcurrentDeposit" time="0.001"/>
  <testcase classname="edu.mu.examplefortesting.tests.TransferTest" name="testConcurrentDeposit" time="0.001"/>
  <testcase classname="edu.mu.examplefortesting.tests.FailingTests" name="testBatchTransferWrongAssertion" time="0.001">
    <failure type="java.lang.AssertionError" message="Expected from.balance = 100.0, but got 50.0 expected [100.0] but found [50.0]">
      <![CDATA[java.lang.AssertionError: Expected from.balance = 100.0, but got 50.0 expected [100.0] but found [50.0]
at edu.mu.examplefortesting.tests.FailingTests.testBatchTransferWrongAssertion(FailingTests.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testBatchTransferWrongAssertion -->
  <testcase classname="edu.mu.examplefortesting.tests.FailingTests" name="testWithdrawNegativeAmount" time="0.0">
    <failure type="org.testng.TestException" message="
Method FailingTests.testWithdrawNegativeAmount()[pri:0, instance:edu.mu.examplefortesting.tests.FailingTests@639c2c1d] should have thrown an exception of type class java.lang.IllegalArgumentException">
      <![CDATA[org.testng.TestException: 
Method FailingTests.testWithdrawNegativeAmount()[pri:0, instance:edu.mu.examplefortesting.tests.FailingTests@639c2c1d] should have thrown an exception of type class java.lang.IllegalArgumentException
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- testWithdrawNegativeAmount -->
  <testcase classname="edu.mu.examplefortesting.tests.InterestTest" name="testAddInterest" time="0.001"/>
  <testcase classname="edu.mu.examplefortesting.tests.InterestTest" name="testAddInterest" time="0.203"/>
  <testcase classname="edu.mu.examplefortesting.tests.TransferTest" name="testBatchTransfer" time="0.001"/>
  <testcase classname="edu.mu.examplefortesting.tests.AccountTest" name="disabledTest" time="0.0">
    <ignored/>
  </testcase> <!-- disabledTest -->
</testsuite> <!-- AllTests -->
